handlers/routes.py

```diff
@@ -425,7 +434,12 @@ def create_download_pdf_auth(repo_url, userjson, email, main_tex="main.tex", opt
             file_full_path = filesdir + "/" + new_main_tex.split(".")[0] + ".pdf"
             run_git_rev_parse = subprocess.check_output(rev_parse, shell=True, cwd=filesdir)
             complete_hash = get_hash([timestamp, email], [run_git_rev_parse.decode('UTF-8')])
-            run_latex_result = subprocess.call("texliveonfly --compiler=pdflatex "+ new_main_tex , shell=True, cwd=filesdir)
+            run_latex_result = subprocess.call("texliveonfly --compiler=latex " + new_main_tex, shell=True,
```

Comment:
Add the url of the reason of it

Reply:
Here are some of the reasons, as a resume, this has to be run 3 times because the output files on the first run are needed for the second (bibliography) and then the third execution will join everything
references:
 https://www.latex-tutorial.com/tutorials/bibtex/
https://stackoverflow.com/questions/2461905/compiling-latex-bib-source